index.js

```diff
@@ -41,7 +41,13 @@ var executable = 'geckodriver';
 var downloadOptions = {}
 var proxy = process.env.HTTPS_PROXY || process.env.HTTP_PROXY || process.env.https_proxy || process.env.http_proxy || null;
 if (proxy !== null) {
-  downloadOptions.agent = new proxyAgent(proxy);
+  downloadOptions.agent = {
```

Comment:
Not sure if it is necessary to switch proxy agent to tunnel 
But in my try I simply changed this line and it works.
```
  downloadOptions.agent = { https: new proxyAgent(proxy) };

```

Reply:
Thanks @pillsilly, do you mind sending a PR?

Reply:
> Thanks @pillsilly, do you mind sending a PR?

😃 here it is https://github.com/vladikoff/node-geckodriver/pull/92